src/components/Button/Button.styles.ts

```diff
@@ -2,13 +2,114 @@ import styled from 'styled-components';
 
 import colors from '@Styles/colorsForJs.module.scss';
 
-import {ButtonColorMap, StyledButtonProps} from './Button';
+import {ButtonColorMap, ButtonSizeMap, StyledButtonProps} from './Button';
 
 /**
  * Styled container for the Button to configure according to props
  */
-export const StyledButtonContainer = styled.button<StyledButtonProps>(({variant, fullWidth, iconPosition}) => {
-    const colorMap: ButtonColorMap = {
+export const StyledButtonContainer = styled.button<StyledButtonProps>(({variant, fullWidth, iconPosition, size = 'md', inverted, destructive, width}) => {
+    // styles depending on different button sizes
+    const increaseButtonSize: ButtonSizeMap = {
```

Comment:
This variable name doesn't seem appropriate.

Reply:
Modified